src/Host/Broker/Impl/Sessions/SessionsController.cs

```diff
@@ -35,16 +35,8 @@ public class SessionsController : Controller {
                 return Task.FromResult<IActionResult>(new ApiErrorResult(BrokerApiError.NoRInterpreters));
             }
 
-            SecureString securePassword = null;
-            string password = User.FindFirst(Claims.Password)?.Value;
-            if (password != null) {
-                securePassword = new SecureString();
-                foreach (var ch in password) {
-                    securePassword.AppendChar(ch);
-                }
-            }
-
             string profilePath = User.FindFirst(Claims.RUserProfileDir)?.Value;
+            var password = User.FindFirst(Claims.Password)?.Value.ToSecureString();
```

Comment:
,,, although `SecureString` provides the same encryption guarantees. So we might as well just have our own static dictionary of IDs to `SecureString`, and use IDs for claims. That way, it  all gets cleaned up when process exits, and has no potential to spill over to other apps. Also, cred manager requires a logon session, it seems, which might be bad if we want to make broker a service in the future?